### PR TITLE
Fix the specs

### PR DESCRIPTION
**Pull request by bumi**
ooops, this is embarrising ;)

### CODE REVIEW

File: test_api.py [hunk 1]
```diff
@@ -114,12 +113,13 @@ def test_bulk(self):
         assert rows2[0][3] == 17
 
         power = list(api.influx.query('select value from power').get_points(measurement='power'))
-        assert power[0]['value'], 1437
-        assert power[1]['value'], 1137.42
+        assert power[0]['value'] == 17
+        assert power[1]['value'] == 16
```
Comment (bumi) [inline test_api.py:124]: @alptilev can you look at the bulk test and verify the expectations? 
from the request data this is fine... can you confirm?

Comment (jlark) [inline test_api.py:124]: Sorry was OOF yesterday, Will do this today

**Comment by jlark** [comment]
can you fix conflicts?

**Comment by bumi** [comment]
done

**Pull request merged**